src/main/java/banking/CardValidator.java

```diff
@@ -0,0 +1,62 @@
+package banking;
+
+import banking.model.Card;
+
+public class CardValidator {
```

Comment:
This class has a minimum of two responsibilities - Card generation and validation. 
Can you do refactoring?

Reply:
fixed